templates/web/prod/User/Add.html.twig

```diff
@@ -108,11 +108,13 @@
             }
         });
     };
-
-    $addUserForm.find('.geoname_field').geocompleter({
-        "server": "{{ app['geonames.server-uri'] }}",
-        "limit": 40
-    });
+    geolink = '{{ app['geonames.server-uri'] }}';
+    if(geolink.length > 0) {
+        $addUserForm.find('.geoname_field').geocompleter({
+            "server": "{{ app['geonames.server-uri'] }}",
+            "limit": 40
+        });
+    }
```

Comment:
since app['geo...'] is a constant from the twig point of view, it's better to do a "twig:if" and not generate the useless js code at all.

Reply:
It is corrected

Reply:
the js code is still generated with "server": "".
The "if" could wrap the whole js code, not the class.

Reply:
This part of code is from the 'geonames-server-jquery-plugin/jquery.geonames.js' whitch is a library called via gulp so i can't edit it. Unless if we will call this file separately ?